core/cap-0038.md

```diff
@@ -7,10 +7,10 @@ Working Group:
     Owner: Jonathan Jove <@jonjove>
     Authors: Jonathan Jove <@jonjove>, Siddharth Suresh <@sisuresh>
     Consulted: OrbitLens <@orbitLens>, Nikhil Saraf <@nikhilsaraf>, Tamir Sen <@tamirms>, Phil Meng <@phil-stellar>, Leigh McCulloch <@leighmcculloch>, Nicolas Barry <@monsieurnicolas>
-Status: Accepted
+Status: Final
```

Comment:
@jonjove The status also needs updating in the CAP table in the readme if we're changing its status here.

Reply:
@leighmcculloch fixed that and several other status issues in #1085